src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/HelloWorldController.java

```diff
@@ -18,4 +20,18 @@ public String sayHello() {
         return helloWorldService.sayHello();
     }
 
-}
+    @DeleteMapping("{idWallet}")
+    @ResponseStatus(HttpStatus.ACCEPTED)
+    void deleteSingleWallet(@PathVariable @Min(1) Long idWallet) {
```

Comment:
You should create proper controller for this issue.

Reply:
Method ** deleteSingleWallet** should return proper value in every case. The type of the result is ResponseEntity.

Reply:
In this project (now or if the future) exceptions are going to be handled by Exception Advice.

Reply:
this has been corrected